src/helpers/web.ts

```diff
@@ -15,9 +12,15 @@ import {
   UploaderEnvs,
   UploaderInputs,
 } from '../types'
-import { info } from './logger'
+import { UploadLogger, info } from './logger'
 import { addProxyIfNeeded } from './proxy'
 
+import { promisify } from 'util'
+import { setTimeout } from 'timers'
+
+const maxRetries = 3
```

Comment:
I think we can bump this up, or at least increase `baseBackoffDelayMs`. We are waiting only a couple seconds in total

Reply:
I've updated it to retry after 1, 2, 4 and 8 seconds (so total wait of 15s + the request times)

Do you think that's enough?

Reply:
let's try it for now and see what happens!